ParadoxLabs_Authnetcim/templates/customer/form/cc.phtml

```diff
@@ -0,0 +1,615 @@
+                    <?= $block->escapeHtmlAttr($this->helper(\Magento\Customer\Helper\Address::class)->getAttributeValidationClass('postcode')) ?>
+                "
+            >
+        </div>
+        <div
+            class="
+                select
+                select--native
+                country
+                required
+            "
+        >
+            <label
+                class="label"
+                for="country"
+            >
+                <?= $block->escapeHtml(__('Country')) ?>
+            </label>
+            <?php
+                $countries = $block->getCountryCollection()
+                    ->setForegroundCountries($block->getTopDestinations())
+                    ->toOptionArray();
+                $countryValue = $block->getAddress()->getCountryId() ?? $block->getCountryId();
+            ?>
+            <select
+                id="country"
+                name="billing[country_id]"
+                class="select__field required-entry"
+                title="<?= $block->escapeHtmlAttr(__('Country')) ?>"
+                data-validate="{'validate-select':true}"
+                data-mage-init='{"select": {}}'
+            >
+                <?php foreach ($countries as $country): ?>
+                    <option
+                        value="<?= $country['value'] ?>"
+                        <?php if ($country['value'] === $countryValue): ?>
+                            selected="selected"
+                        <?php endif ?>
+                    >
+                        <?= $block->escapeHtml($country['label']) ?>
+                    </option>
+                <?php endforeach ?>
+            </select>
+        </div>
+    </fieldset>
+    <fieldset
+        class="
+            fieldset
+            payment
+            items
+            ccard
+            <?= $code ?>
+        "
+    >
+        <legend class="fieldset__legend">
+            <?= $block->escapeHtml(__('Credit Card Information')) ?>
+        </legend>
+
+        <div
+            class="
+                input
+                type
+                required
+            "
+        >
+            <?php $_ccType = $block->getCard()->getAdditional('cc_type') ?>
+            <ul class="credit-card-types list list--horizontal">
+                <?php foreach ($block->getCcBlock()->getCcAvailableTypes() as $_typeCode => $_typeName) : ?>
+                    <li
+                        class="
+                            list__item
+                            item<?php if ($_typeCode == $_ccType) : ?>
+                            _active<?php endif; ?>
+                        "
+                        data-type="<?= $block->escapeHtmlAttr($_typeCode); ?>"
+                    >
+                        <div class="lazyload-wrapper">
+                            <img
+                                src="<?= $block->getViewFileUrl('Magento_Payment::images/cc/' . strtolower($_typeCode) . '.png'); ?>"
```

Comment:
if lazyload is used, shouldn't be `data-src` attribute used?

Reply:
You've right but we don't need lazy loading here, I'll remove it